src/Bridge/Doctrine/Orm/QueryAwarePartialPaginatorInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Bridge\Doctrine\Orm;
+
+use ApiPlatform\Core\DataProvider\PartialPaginatorInterface;
+use Doctrine\ORM\Query;
+
+interface QueryAwarePartialPaginatorInterface extends PartialPaginatorInterface
+{
+    public function getQuery(): Query;
```

Comment:
```suggestion
    /**
     * Gets the Query object that will actually be executed.
     *
     * This is necessary as some options could only be set on the Query object itself.
     */
    public function getQuery(): Query;
```

Reply:
@desmax Please add the documentation as I've provided.

Reply:
@alanpoulain updated, though I don't understand why last two words have to be on a new line.

Reply:
It's just a hard line break at 80 characters, for readability.

Reply:
Sorry, I think you've misunderstood something. The PHPDoc should be on the interface, not the implementation.

Reply:
@teohhanhui all updated, based it on master